crates/store/re_remote_store_types/src/lib.rs

```diff
@@ -0,0 +1,339 @@
+//! This crate contains generated types for the remote store gRPC service API.
+//! Generation is done using the `re_remote_store_types_builder` crate.
+//!
+//! We want clear separation between 'internal' types and gRPC types and don't want
+//! to use gRPC types in the rerun viewer codebase. That's why we implement all the
+//! necessary conversion code (in the form of `From` and `TryFrom` traits) in this crate.
+//!
+
+// Ignoring all warnings for the auto-generated code.
+#![allow(clippy::doc_markdown)]
+#![allow(clippy::derive_partial_eq_without_eq)]
+#![allow(clippy::enum_variant_names)]
+#![allow(clippy::unwrap_used)]
+#![allow(clippy::wildcard_imports)]
+#![allow(clippy::manual_is_variant_and)]
```

Comment:
You should be able to apply these only to the `mod _v0` below.

Reply:
makes sense, updated